test/test_s3_utils.py

```diff
@@ -908,12 +1006,26 @@ class MyS3(MockBotoS3Client):
 
 def test_env_manager_global_env_bucket_name():
 
-    # These tests expect to be run in an environment that does not have these buckets bound globally.
-    assert os.environ.get('GLOBAL_ENV_BUCKET') is None
-    assert os.environ.get('GLOBAL_BUCKET_ENV') is None
+    # Now that we're containerized, there is always something in GLOBAL_ENV_BUCKET
+    #
+    # # These tests expect to be run in an environment that does not have these buckets bound globally.
+    # assert os.environ.get('GLOBAL_ENV_BUCKET') is None
+    # assert os.environ.get('GLOBAL_BUCKET_ENV') is None
+    #
+    # This is the rewrite:
+
+    global_env_bucket_for_testing = os.environ.get('GLOBAL_ENV_BUCKET')
+    assert global_env_bucket_for_testing  # probably it's 'foursight-envs', but let's be generic
```

Comment:
It needs to be `foursight-envs` IMO. If it's not you're trying to run utils tests in the wrong account and will get many failures. This is the type of thing that should be tested before kicking any other tests.